### DISCUSSION TITLE
What is "Bypass Emulated Keyboard" for in the hotkeys?

### DISCUSSION THREAD

**Discussion by kathyrollo** [flyinghead/flycast#1722] [Q&A] [answered]
Hi @flyinghead ,

I haven't been tinkering with my control settings for a while, what's the (relatively) new hotkey "Bypass Emulated Keyboard" for?

Thanks!

**Comment by flyinghead** [comment, marked as answer]
It's only useful when you use an emulated dreamcast keyboard. In this case, all keyboard keys are sent to the game. Holding down the "Bypass" key allows you to send keys to the emulator instead, for example to pause the game (Bypass+TAB).

**Reply by kathyrollo** [reply]
I see, should be handy for The Typng of the Dead.

Your answer is definitely a useful tooltip in the emulator like some other configs have for improved UX, as the purpose and usage of the bypass key isn't immediately obvious. :)

Thanks for the speedy help as always @flyinghead ! Marked as answer. 👍
